src/main/java/core/basesyntax/service/impl/StorageTransactionsImpl.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.StorageTransactions;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class StorageTransactionsImpl implements StorageTransactions {
+    @Override
+    public List<String[]> convertFileIntoList(File inputFile) {
+        List<String> inputData;
+        try {
+            inputData = Files.readAllLines(inputFile.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file " + inputFile);
+        }
+        return inputData.stream()
+                .map(i -> i.split(","))
+                .collect(Collectors.toList());
+    }
```

Comment:
in this step you can only safe data to string list. Do not split ","
This logic must be when you will form a list of fruitoperations